.docker/base/start-agent.sh

```diff
@@ -1,22 +1,27 @@
 #!/bin/bash
+#Prepare everest-ci and everest-logs repositories; adding environment variable
+#PAT and agent values should be provided in the docker run command
+
+cd $MYHOME
+#Add environment variable for slack notifications
+export SLACK_FSTAR_WEBHOOK=https://hooks.slack.com/services/${PAT_SLACK}
```

Comment:
Does docker also substitute the environment variables provided on the command-line within scripts added with `ADD`?